signals/attachments/attachment_content_type_delivery_status.yml

```
@@ -0,0 +1,4 @@
+name: "Attachment: Content Type Message Delivery Status Notification"
```

```suggestion
name: "Attachment: Message Delivery Status Notification Content Type"
```
just to have it the same way as the "EML Content Type" signal!